docs/components/stepper/test.html

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Please remove Stepper test page. Given that a stepper is just a _styled_ Accordion, its behavior is exactly the same and testing it would add no extra value on top of Accordion tests.